src/main/java/frc/robot/Robot.java

```diff
@@ -89,4 +98,20 @@ public void teleopPeriodic() {
       startingRotations = (Drive.getRightPosition() + Drive.getLeftPosition() /2);
     }
   }
+  public static void Shiftersint (boolean isShifted){
```

Comment:
Should this not be in a class of it's self? Such that you can call something like Shifters.set(HIGH_GEAR) or something like that?

Reply:
it could, but it pertains to the drive train so I thought it would fit better within drive itself